csharp/src/Ice/Internal/ServantManager.cs

```diff
@@ -103,7 +97,7 @@ public Ice.Object removeDefaultServant(string category)
         {
             Debug.Assert(_instance != null); // Must not be called after destruction.
 
-            Ice.Object obj = null;
+            Ice.Object? obj = null;
             _defaultServantMap.TryGetValue(category, out obj);
```

Comment:
Inline the declaration
```suggestion
            _defaultServantMap.TryGetValue(category, out Object? obj);
```

Reply:
Fixed